ingeniamotion/fsoe.py

```diff
@@ -115,8 +117,9 @@ def is_sto_active(self) -> bool:
         fsoe_command = int.from_bytes(self.safety_slave_pdu_map.items[0].raw_data_bytes, "little")
```

Comment:
You should not need to check the state of the the state machine or cehck if is in failsafe...
If it's not in process data safe inputs on the dictionary must be set to failsafe
https://novantamotion.atlassian.net/browse/INGK-922

Reply:
With this PR: https://bitbucket.org/novantamotion/fsoe_master_python/pull-requests/39/overview
I think this should be enough
````
def is_sto_active(self) -> bool:
        """Check the STO state.

        Returns:
            True if the STO is active. False otherwise.

        """
        sto_command = self.__master_handler.dictionary.get(self.STO_COMMAND_UID)
        if not isinstance(sto_command, bool):
            raise ValueError(f"Wrong value type. Expected type bool, got {type(sto_command)}")
        return sto_command
````